include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Yes, but for

1) We are asking the user to add a CppUTest include to *their* Makefile or project file. I would like to avoid that as much as possible as most Makefile/project files already have way too many #defines.

2) Unlikely. If we don't have the #ifdef and someone would try to use the macro, then they'll need to actually define a long long. If they do so, it would give an error. Unless they try to pass it with a fixed int constant, then we can add a constructor to make a proper run-time error.